piccolo/apps/asgi/commands/templates/app/_xpresso_app.py.jinja

```diff
@@ -0,0 +1,111 @@
+import typing as t
+from contextlib import asynccontextmanager
+
+from piccolo.engine import engine_finder
+from piccolo.utils.pydantic import create_pydantic_model
+from piccolo_admin.endpoints import create_admin
+from starlette.staticfiles import StaticFiles
+from xpresso import App, FromJson, FromPath, HTTPException, Operation, Path
+from xpresso.routing.mount import Mount
+
+from home.endpoints import home
+from home.piccolo_app import APP_CONFIG
+from home.tables import Task
+
+TaskModelIn: t.Any = create_pydantic_model(table=Task, model_name="TaskModelIn")
+TaskModelOut: t.Any = create_pydantic_model(
+    table=Task, include_default_columns=True, model_name="TaskModelOut"
+)
+
+
+async def tasks() -> t.List[TaskModelOut]:
+    return await Task.select().order_by(Task.id)
+
+
+async def create_task(task_model: FromJson[TaskModelIn]) -> TaskModelOut:
+    task = Task(**task_model.__dict__)
```

Comment:
Just for my knowledge, is there any particular reason to do `task_model.__dict__` instead of `task_model.dict()`? Same with `for key, value in task_model.__dict__.items()` [below](https://github.com/piccolo-orm/piccolo/blob/6566deb7a7c5d8868893291a2547084091c65b6a/piccolo/apps/asgi/commands/templates/app/_xpresso_app.py.jinja#L38).

Reply:
You're right - it should be `dict()`.